pims/display.py

```diff
@@ -420,14 +431,19 @@ def plot_to_frame(fig, width=512, close_fig=False, bbox_inches=None):
     from pims import Frame
     if isinstance(fig, mpl.axes.Axes):
         fig = fig.figure
+    if fig_size_inches is not None:
```

Comment:
It was already, I put it in the docs. It can be a len 2 tuple or None, in which case the figsize is unchanged.